dataflow/flex-templates/streaming_beam_sql/metadata.json

```diff
@@ -7,7 +7,7 @@
       "label": "Pub/Sub input subscription.",
       "helpText": "Pub/Sub subscription to read from.",
       "regexes": [
-        "[-_.a-zA-Z0-9]+"
+        "(?!goog)[a-zA-Z][-_.~+%a-zA-Z0-9]{2,}"
```

Comment:
You might wish to explain and put a pointer to where they can find more information on this.

Reply:
Those are for user-defined inputs, in this case it happened to be Pub/Sub which has very specific validation rules, but it could be whatever the user wants to validate for in their own pipelines. What kind of information were you thinking about?

Reply:
I was thinking that you might wish to link put a  comment w/ a link to the docs at a minimum.  Since `(?!goog)` looks a bit like magic.  Or a comment that says something like `// (?!goog) makes the regular expression syntax behave like ...`

Reply:
Or is `goog` a prefix?   https://regex101.com/

Reply:
It is negative lookahead on a prefix, Pub/Sub doesn't allow topics or subscriptions starting with `goog`.

Reply:
I've included some links in the README